Dockerfile

```diff
@@ -1,20 +1,13 @@
 FROM ubuntu:16.04
 
-LABEL maintainer="zyxep"
```

Comment:
Removing this as it is deprecated https://docs.docker.com/engine/reference/builder/#label

Reply:
Label is not deprecated, maintainer is.

Reply:
Will bring this back.